development/decision_process.rst

```diff
@@ -0,0 +1,182 @@
+######################################
+Discussion and Decision Making Process
+######################################
+
+This page describes how decisions are made in Data Management.
+Generally, DM teams and staff have agency to implement non-controversial work (see § :ref:`decision-making-empowerment`).
```

Comment:
"agency" is a perhaps-trendy but rather jargon-y word.  I'd prefer not to use it.